clusters/build-clusters/01_cluster/openshift-monitoring/cluster-monitoring-config_configmap.yaml

```diff
@@ -19,3 +19,8 @@ data:
          resources:
            requests:
              storage: 10Gi
+    thanosQuerier:
+      # workaround before we have a fix for https://bugzilla.redhat.com/show_bug.cgi?id=1906496
+      resources:
+        limits:
+          memory: 500Mi
```

Comment:
maybe let's set this to 1GB, I think we are indeed idling at ~600MB

Reply:
Done.